python/ray/util/queue.py

```diff
@@ -212,6 +215,14 @@ def get_nowait_batch(self, num_items: int) -> List[Any]:
 
         return ray.get(self.actor.get_nowait_batch.remote(num_items))
 
+    def shutdown(self) -> None:
+        """Terminates the underlying QueueActor and release all of its
+        resources.
```

Comment:
Top line of a doc should not wrap onto a second line (https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)